doc/adding-your-service.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "Adding your service to the mesh"
+docpage = true
+[menu.docs]
+  parent = "adding-your-service"
++++
```

Comment:
I'm a bit torn on adding the hugo frontmatter section to these docs, since it interferes with the GitHub rendering:

![image](https://user-images.githubusercontent.com/9226/36709681-f682ce6c-1b2d-11e8-94de-424e343bc7b9.png)

It might be a bit more work when building the site, but I could envision a setup where we just use a markdown H1 tag here, and then convert that tag to frontmatter when importing the doc into the site repo. Something like:

```
# Adding your service to the mesh

In order for your service to take advantage of Conduit, it needs to be added
...
```

That would certainly look prettier on GitHub, and it hides the conduit.io implementation details.